src/packages/bem/__tests__/bem.spec.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import bem from '../bem';
+import { Button, ButtonStateless } from './dummy-components';
```

Comment:
But it's not mocks, it's real components

Reply:
they are only used in tests, hence they 'aren't real'

Reply:
As for me `mock` is something that
1. Mocks dependency
2. Can be traceable (like we should be able to get a number of calls of a methio...)

In terms of jest `__mocks__` is a folder where you supposed to put mocked implementation of your class so that other modules that depends on your module don't need mock it every time (more info here https://jestjs.io/docs/en/manual-mocks)

In this case dummy-component is not a dependency, it's an input data for bem function